.github/workflows/continuous-integration.yml

```diff
@@ -30,6 +30,7 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
I dunno, I'd say we can keep fail-fast (if something breaks on nodejs 14 we don't need nodejs 16 or 18 to pass)

Reply:
The reason I prefer fail-fast false is because I want to know whether Node 16 and Node 18 actually also work or if they also fail. With fail-fast set to true you are left completely in dark and you may end up looking for a fix in completely the wrong direction.